examples/reference/panes/Plotly.ipynb

```diff
@@ -33,22 +33,25 @@
     "* **``object``** (object): The Plotly `Figure` or dictionary object being displayed.\n",
     "* **``config``** (dict): Additional configuration of the plot. See [Plotly configuration options](https://plotly.com/javascript/configuration-options/).\n",
     "\n",
-    "### Callbacks\n",
+    "### Events\n",
     "\n",
-    "* **``click_data``** (dict): Click callback data\n",
-    "* **``clickannotation_data``** (dict): Clickannotation callback data\n",
-    "* **``hover_data``** (dict): Hover callback data\n",
-    "* **``link_figure``** (bool): Attach callbacks to the Plotly figure to update output when it is modified in place.\n",
-    "* **``relayout_data``** (dict): Relayout callback data\n",
-    "* **``restyle_data``** (dict): Restyle callback data\n",
-    "* **``selected_data``** (dict): Selected callback data\n",
-    "* **``viewport``** (dict): Current viewport state, i.e. the x- and y-axis limits of the displayed plot.\n",
+    "* **``click_data``** (dict): Click event data from `plotly_click` event.\n",
+    "* **``clickannotation_data``** (dict): Clickannotation event data from `plotly_clickannotation` event.\n",
+    "* **``hover_data``** (dict): Hover event data from `plotly_hover` and `plotly_unhover` events.\n",
+    "* **``relayout_data``** (dict): Relayout event data from `plotly_relayout` event\n",
+    "* **``restyle_data``** (dict): Restyle event data from `plotly_restyle` event\n",
+    "* **``selected_data``** (dict): Selected event data from `plotly_selected` and `plotly_deselect` events.\n",
+    "* **``viewport``** (dict): Current viewport state, i.e. the x- and y-axis limits of the displayed plot. Updated on `plotly_relayout`, `plotly_relayouting` and `plotly_restyle` events.\n",
     "* **``viewport_update_policy``** (str, default = 'mouseup'): Policy by which the viewport parameter is updated during user interactions\n",
     "    * ``mouseup``: updates are synchronized when mouse button is released after panning\n",
     "    * ``continuous``: updates are synchronized continually while panning\n",
     "    * ``throttle``: updates are synchronized while panning, at intervals determined by the viewport_update_throttle parameter\n",
     "* **``viewport_update_throttle``** (int, default = 200, bounds = (0, None)): Time interval in milliseconds at which viewport updates are synchronized when viewport_update_policy is \"throttle\".\n",
     "\n",
+    "### Other\n",
+    "\n",
+    "\n",
+    "* **``link_figure``** (bool): Attach callbacks to the Plotly figure to update output when it is modified in place.\n",
```

Comment:
I don't understand this one. It would be nice to document. If you have time please provide time and explanation.

Reply:
Understood. Example added.

Reply:
Ah, was just in the process of editing.

Reply:
I will combine the "Patching" and "updating in place" sections as they are really just alternative approaches to do the same thing.